src/components/Button/ButtonTypes.ts

```diff
@@ -37,6 +37,10 @@ export type BaseProps = {
    * 後方配置のアイコン
    */
   suffixIcon?: 'default' | ReactNode;
+  /**
+   * ローディング状態を示す
+   */
+  loading?: boolean;
```

Comment:
The loading prop should only be <button>. It would be good to add it to `OnlyButtonProps`.

Reply:
I know, but I though there is some usecases you want loading status while routing.
like: https://github.com/ubie-inc/ofro/issues/4108

Reply:
@kodai3 CC: @takanorip 

In my layman's opinion, if the process occurs after clicking a link, then the button is correct, not the link 🤔.

I can understand that there are useful cases.
I guess these issues come up because JavaScript is reimplementing the browser😇.

You may want to apply Loading to the Link as well, to give priority to other issues

If you want to apply Loading to Link as well, please add the implementation to src/components/Button/LinkButton.tsx as well!
(Incidentally, LinkButton avoids implementing the disabled attribute.)

Reply:
Since a link is a link, there shouldn't be any issues with multiple clicks. However, if any problems arise, I believe using a Button would be more appropriate.
I don't want to avoid adding unnecessary properties, so it would be helpful to carefully examine the use case...

Reply:
> If you want to apply Loading to Link as well, please add the implementation to src/components/Button/LinkButton.tsx as well!

Oh, I didn't see that `LinkButton` is totally different implementation. 😇

Let's leave it added only on `Button` and see if we really want on `LinkButton`